.github/workflows/split_monorepo.yaml

```diff
@@ -1,23 +1,131 @@
 name: gitsplit
 on:
-  push:
-    branches:
-      - main
-      - split
   release:
     types: [published]
-  create:
-  workflow_dispatch:
 
 jobs:
-  gitsplit:
+  split-hook-dd-trace:
```

Comment:
I wonder if you could make this a matrix. I played around with something similar here.

https://github.com/beeme1mr/github-actions-experiments/blob/main/.github/workflows/js-test.yaml

Happy to collaborate with you if you're interested!

Reply:
The issue I see is that [a matrix strategy is defined within a single job](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#using-a-matrix-strategy), but each of these are defined as separate jobs.

I do want to eventually simplify this, but I really want to prove it satisfies the needs of this repository first. Theoretically `git-filter-repo` should work, but so did `gitsplit` at the time :slightly_smiling_face:

Reply:
We should take a look at other approaches to this if all of this ends up actually working though. The entirety of each job would be be defined best as part of a single Action which would simplify a ton of this. `gitsplit` had a config file that would also be a good approach to this.

Reply:
I am going to merge this just for the sake of testing that the `git-filter-repo` strategy works, and then we can work on making this more clean and DRY code.